packages/next/lib/constants.ts

```diff
@@ -73,7 +73,7 @@ export const ESLINT_PROMPT_VALUES = [
     },
   },
   {
-    title: 'None',
+    title: 'Cancel',
     config: null,
   },
-]
+] as any
```

Comment:
Can you use `as const` instead of any?

Reply:
Removed the type altogether. Included `any` after I started seeing type assertion issues but just used a [non-null assertion](https://github.com/vercel/next.js/pull/26584/files#diff-24b42de85d88fa519f680943c6fb26537eb951b770d9c1f3e8659d472fca37c7R287) instead